src/test/java/com/google/sps/data/DatastoreClientTest.java

```diff
@@ -0,0 +1,155 @@
+package com.google.sps.data;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+//import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.Assert;
+import org.junit.Test;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import java.util.Date;
+import java.util.Optional;
+import com.google.sps.data.Attendee;
+import com.google.sps.data.AttendeeInterface;
+import com.google.sps.data.Session;
+import com.google.sps.data.SessionInterface;
+import com.google.sps.data.Instance;
+import com.google.sps.data.InstanceInterface;
+import com.google.sps.data.DatastoreClient;
+import com.google.sps.data.DatastoreClientInterface;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/* Class that test the methods in DatastoreClient. */
+public class DatastoreClientTest  {
+  private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testAttendeeIsInsertedToDatastore() {
+    // Tests that object is inserted to datastore.
+    DatastoreClientInterface datastoreClient = new DatastoreClient();
```

Comment:
Do you need a separate client for each test? If not, you can make this a private member variable like you did with LocalServiceTestHelper.